tensorly/decomposition/candecomp_parafac.py

```diff
@@ -4,8 +4,6 @@
 import tensorly as tl
```

Comment:
Should we use relative import here too?

Reply:
Won't it not use the kruskal_to_tensor in the correct backend otherwise?